gatsby/content/docs/free-small-matrix-server.mdx

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
Looks good on http://localhost:8000/docs/guides/free-small-matrix-server ✅

We could also add a link to it from http://localhost:8000/docs/guides/ which seems to be manually curated https://github.com/matrix-org/matrix.org/blob/9255200369fab9fbff211aa871cf993773139e70/gatsby/src/pages/docs/guides.js#L138-L156